index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Survay Form</title>
+  </head>
+  <body>
+    <h1 id="title">Survay Form</h1>
+    <p id="description">
+      This is an assignment task from freecodecamp to get certificate of front
+      end designer
+    </p>
```

Comment:
I think it would be better to enclose this part in a section. It will make your code more professional because it is the best practice to use semantic tags.